addon/stories/form.stories.js

```diff
@@ -0,0 +1,75 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export const form = (args) => {
+  return {
+    template: hbs`
+    <form>
+      <PixInput @id='firstName' @label='Prénom' @errorMessage={{genericErrorMessage}} />
+      <br>
+
+      <PixMultiSelect
+        @title="Votre notation en étoiles..."
+        @id="form__pix-mutli-select"
+        @label="A quel point aimez vous Pix-UI ?"
+        @onSelect={{onSelect}}
+        @selected={{selected}}
+        @options={{options}} as |star|
+      >
+        <PixStars @count={{star.value}} @total={{star.total}} />
+      </PixMultiSelect>
+      <br><br>
+
+      <PixSelect
+        @id="form__searchable-pix-select"
+        @label="Votre fruit préféré est : "
+        @options={{selectOptions}}
+        @isSearchable={{true}}
+        @isValidationActive={{true}}
+        placeholder='Fraises, Mangues...'
+      />
+      <br>
+
+      <PixTextarea
+        @id="form__pix-textarea"
+        @value=""
+        @maxlength=200
+        @label="Un commentaire ?"
+        @errorMessage={{genericErrorMessage}} />
+      <br>
+
+      <div class="pix-form__actions">
+        <PixButton @triggerAction={{cancel}} @backgroundColor="transparent-light" @isBorderVisible={{true}}>
+          Annuler
+        </PixButton>
+        <PixButton @type="submit">Envoyer mes réponses</PixButton>
```

Comment:
Suggestion : 

```suggestion
        <PixButton @type="submit" @triggerAction={{action triggerAction}}>Envoyer mes réponses</PixButton>
```
Je me dit que c'est intéressant de l'ajouter car il est obligatoire et comme on peut voir ce code sur Storybook en cliquant sur Show Code, on a quelque chose de fonctionnel :)

Reply:
Si on met un bouton de type "submit" dans un `<form>`, alors `@triggerAction` n'est pas obligatoire. On va faire porter la soumission du formulaire sur `<form {{on "submit" this.submit}} />`

Reply:
En version "submit" le triggerAction n'est pas obligatoire (car souvent on met l'action submit dans la balise <form>). Du coup je trouve que c'est moins représentatif de l'utilisation qu'on en fait. Mais le PixButton "Annuler" l'a lui le triggerAction :)